chroma-manager/tests/integration/core/api_testcase_with_test_reset.py

```diff
@@ -981,7 +981,7 @@ def cleanup_zpools(self):
         ]
 
         def wipe(x):
-            return 'wipefs -a {0} && parted {0} mklabel gpt'.format(x)
```

Comment:
This is working ok with SSI, maybe we should only change it for EFS.

Reply:
agreed, would have been nice if we could have made it consistent across both